frontend/src/static/js/utils/urls.ts

```diff
@@ -43,6 +43,10 @@ export function getWPTMetricView(location: {search: string}): string {
   return getQueryParam(location.search, 'wpt_metric_view');
 }
 
+export function getFeaturesLaggingFlag(location: {search: string}): boolean {
```

Comment:
Thanks for this helper. In the future, I think we should use this pattern for this and the usage chart. In the future, we could have an enumeration of feature flags and then call this method.:

```ts
enum ClientSideFeatureFlag {
  showUsageChart = 'show_usage_chart',
  showFeaturesLagging = 'show_features_lagging'
}



export function isFeatureEnabled(location: {search: string}, flag: ClientSideFeatureFlag): boolean {
}

```

Reply:
No action needed for this PR on this.

Reply:
ack. Thanks!